extensions/interactions/CodeRepl/directives/code-repl-interaction.component.html

```diff
@@ -8,7 +8,7 @@
                         [disabled]="!interactionIsActive"
                         [autoFocus]="true"
                         (ngModelChange)="onEditorChange($event)"
-                        [ngModelOptions]="{standalone: true}""
```

Comment:
There is an **unexpected** `"` in code that resulted in errors while writing frontend tests. Hence it has been removed.